packages/components/src/Icon/Icon.test.tsx

```diff
@@ -36,3 +36,13 @@ it("renders star icon with custom color", () => {
   const { container } = render(<Icon name="star" customColor="#f33323" />);
   expect(container).toMatchSnapshot();
 });
+
+it("applies testID prop to svg element", () => {
+  const { getByTestId } = render(<Icon name="star" testID="star-icon" />);
+  expect(getByTestId("star-icon")).toBeInTheDocument();
+});
```

Comment:
Consider also adding a test for the id `star` not being in the document here...

Reply:
Because of my comment [here](https://github.com/GetJobber/atlantis/pull/1927#discussion_r1647809883) we're going to be leaving `data-testid={testID || name}` as is, so I'll continue to test that `star` & `star-icon` can both be used as `testid`'s